rails/benchmarks/support/app_base.rb

```diff
@@ -5,7 +5,6 @@
 require 'active_record'
 require 'sqlite3'
 require 'ffaker'
-require 'jbuilder'
```

Comment:
This one was required for https://github.com/ruby-bench/ruby-bench-suite/pull/17